src/internal/kopia/backup_bases.go

```diff
@@ -116,6 +116,12 @@ func (bb *backupBases) MinBackupVersion() int {
 		}
 	}
 
+	for _, base := range bb.assistBases {
```

Comment:
make this a new function and add it as a separate condition in the if-statement in the backup operation addition from #4561 please. We should keep the info for merge and assist bases separate so we can reason about things easily